MediaAdminBundle/Controller/Api/FolderController.php

```diff
@@ -48,4 +50,25 @@ public function deleteAction($folderId)
 
         return array();
     }
+
+    /**
+     * @param string $siteId
+     *
+     * @Config\Route("/list/tree/{siteId}", name="open_orchestra_api_folder_list_tree")
+     * @Config\Method({"GET"})
+     *
+     * @Config\Security("is_granted('ROLE_ACCESS_MEDIA_FOLDER')")
+     *
+     * @return FacadeInterface
+     */
+    public function listTreeFolder($siteId)
+    {
+        $folders = $this->get('open_orchestra_media.repository.media_folder')->findAllRootFolderBySiteId($siteId);
```

Comment:
The FolderTreeTransformer needs only the root folders to start creating the trees.